### ISSUE TITLE
Getting error in this line evaluator.evaluate(eval_config) , I have tried to convert into python 3 , not getting idea , could you please help me 

### ISSUE THREAD

**Issue by Rishi5583** [scott89/deeplens_eval#5]
File "eval.py", line 35, in <module>
    main()
  File "eval.py", line 29, in main
    evaluator.evaluate(eval_config)
  File "D:\deeplens_eval-master\core\evaluator_interactive.py", line 87, in evaluate
    run_dof_func, run_depth_func = construct_graph(eval_config)
  File "D:\deeplens_eval-master\core\evaluator_interactive.py", line 38, in construct_graph
    depth_320, depth_net = build_depth_resnet.build(image_320_tensor, is_training)
  File "D:\deeplens_eval-master\core\build_depth_resnet.py", line 17, in build
    net = ResNet50({'data': im_resized-0.5}, is_training)
  File "D:\deeplens_eval-master\network\resnet50_lrelu.py", line 7, in __init__
    super(ResNet50, self).__init__(input, trainable)
  File "D:\deeplens_eval-master\network\network.py", line 45, in __init__
    self.setup()
  File "D:\deeplens_eval-master\network\resnet50_lrelu.py", line 11, in setup
    .conv(7, 7, 64, 2, 2, 1, relu=False, name='conv1', padding='VALID')
  File "D:\deeplens_eval-master\network\network.py", line 22, in layer_decorated
    layer_output = op(self, layer_input, *args, **kwargs)
  File "D:\deeplens_eval-master\network\network.py", line 156, in conv
    output = convolve(input, kernel)
  File "D:\deeplens_eval-master\network\network.py", line 148, in <lambda>
    convolve = lambda i, k: tf.nn.conv2d(i, k, [1, s_h, s_w, 1], padding=padding)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 1953, in conv2d
    name=name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_nn_ops.py", line 1161, in conv2d
    data_format=data_format, dilations=dilations, name=name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 788, in _apply_op_helper
    op_def=op_def)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3616, in create_op
    op_def=op_def)
  File "C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 2005, in __init__
    self._traceback = tf_stack.extract_stack()
